src/rest_service.py

```diff
@@ -86,13 +86,19 @@ def do_GET(self):
         if re.search(api_endpoint, self.path):
             query = parse_qs(urlparse(self.path).query)
 
-            if ("experiment_name" in query and "trial_number" in query and hpo_service.instance.containsExperiment(query["experiment_name"][0]) and
-                    query["trial_number"][0] == str(hpo_service.instance.get_trial_number(query["experiment_name"][0]))):
+            if ("experiment_name" in query and "trial_number" in query and hpo_service.instance.containsExperiment(
+                    query["experiment_name"][0]) and
+                    query["trial_number"][0] == str(
+                        hpo_service.instance.get_trial_number(query["experiment_name"][0]))):
+                logger.info("Experiment_Name = " + str(
+                    hpo_service.instance.getExperiment(query["experiment_name"][0]).experiment_name))
+                logger.info("Trial_Number = " + str(
+                    hpo_service.instance.getExperiment(query["experiment_name"][0]).trialDetails.trial_number))
```

Comment:
Should this be logger.info or logger.debug ?

Reply:
It will show these details to differentiate between various experiments. So, that's why I thought it should be "info".